src/aspire/source/image.py

```diff
@@ -673,7 +702,7 @@ def save_images(
                     logger.info(
                         f"Saving ImageSource[{i_start}-{i_end-1}] to {mrcs_filepath}"
                     )
-                    datum = self.images(start=i_start, num=num).data.astype("float32")
+                    datum = self.images[i_start : i_start + num].data.astype("float32")
```

Comment:
Remove `num` and index `i_star:i_end`. Also below.

Reply:
Done